src/test/scala/slick/ext/repo.scala

```diff
@@ -3,9 +3,34 @@ package slick.ext
 import slick.ext.macros._
 
 trait UserRepo {
-  val profile: scala.slick.driver.JdbcProfile
+  //val profile: scala.slick.driver.JdbcProfile
+  val profile = scala.slick.driver.H2Driver
   import profile.simple._
+  import scala.slick.collection.heterogenous._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/jilen-zhang/slickext/file/583337849/issues/source?bid=2273443&fileBranchId=2273469#l9)